i18n/en-US/help/perms.json

```diff
@@ -0,0 +1,40 @@
+{
+    "DESCRIPTION": "Sends a help embed in regards to permissions.",
+    "LEVELS": "Permission Levels",
+    "VIEW_ALL": "There are six possible permission levels to have. Commands for all levels can be seen with `{{prefix}}commands`.",
+    "BLACKLISTED": "» Blacklisted [-1]",
+    "BLACKLISTED_VALUE": "$t(help/perms:BLACKLISTED_VALUE_1)\n$t(help/perms:BLACKLISTED_VALUE_2)\n$t(help/perms:BLACKLISTED_VALUE_3)\n$t(help/perms:BLACKLISTED_VALUE_4)",
+    "BLACKLISTED_VALUE_1": "**Description:** Users with this role are unable to use the bot.",
+    "BLACKLISTED_VALUE_2": "**Inherits From:** N/A",
+    "BLACKLISTED_VALUE_3": "**Setup Command:** `$set edit blacklistrole <role-name>`",
+    "BLACKLISTED_VALUE_4": "**Default Role:** `TypicalBot Blacklisted`",
+    "MEMBER": "» Server Member [0]",
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `»` is `\u00bb`.